.travis.yml

```diff
@@ -95,6 +95,7 @@ jobs:
       env: DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY=no DEPLOY=no EXTRA_ARGS=32_only
       # XXX i#2764: Travis OSX resources are over-subscribed and it can take
       # hours to get an OSX machine, so we skip running PR's for now.
+      # *** TEMPORARY, will get removed before commit ***
```

Comment:
You forgot to remove this?

Reply:
Will remove before commit.

Reply:
Friendly suggestion: please look through the patch yourself beforehand :)